app/src/main/java/org/mozilla/fenix/library/history/HistorySearchDialogFragment.kt

```diff
@@ -52,6 +59,8 @@ class HistorySearchDialogFragment : AppCompatDialogFragment(), UserInteractionHa
     private lateinit var toolbarView: ToolbarView
     private lateinit var awesomeBarView: AwesomeBarView
 
+    private val speechIntent = Intent(RecognizerIntent.ACTION_RECOGNIZE_SPEECH)
+    private var voiceSearchButtonAlreadyAdded: Boolean = false
```

Comment:
```suggestion
    private var voiceSearchButtonAlreadyAdded = false
```
Seems implied but more for consistency since `dialogHandledAction` immediately below doesn't have it

Reply:
That's true, it is implied.  I'll fix.  Thanks